JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -385,16 +397,20 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
 
 .frequentistReliabilityCheckLoadings <- function(dataset, variables) {
   
-  prin <- psych::principal(dataset)
-  idx <- prin[["loadings"]] < 0
-  sidx <- sum(idx)
-  if (sidx == 0) {
-    return()
+  if (ncol(dataset > 2)) {
+    prin <- psych::principal(dataset)
+    idx <- prin[["loadings"]] < 0
+    sidx <- sum(idx)
+    if (sidx == 0) {
+      return("")
+    } else {
+      hasSchar <- if (sidx == 1L) "" else "s"
+      footnote <- gettextf("The following item%s correlated negatively with the scale: %s. ",
+                           hasSchar, paste0(variables[idx], collapse = ", "))
```

Comment:
You should use `sprintf(ngettext())` for this (also in the Bayesian setting), your solution is quite specific for the English language.